tests/include/test/drivers/size.h

```diff
@@ -31,8 +31,10 @@
 
 /** \def TEST_DRIVER_KEY_CONTEXT_BASE_SIZE
  *
- * This macro returns the base size for the key context. It should at a minimum
- * 20 bytes.
+ * This macro returns the base size for the key context. It should enough size
```

Comment:
I don't see any requirement/recommendation in psa-driver-interface.md for a driver to put the base size, key pair size, public key size, symmetric factor, and store public key data in its key contexts, do you ?
To me `base_size` is the size of a driver specific structure that goes along key data in a key context.

Reply:
These constants allow the core to calculate the size of the context from the key parameters in typical use cases where a context contains a commonplace representation of the key, plus some overhead for metadata and encryption.

Reply:
That was how I interpreted the document based on this line.

The "key_context" property in the driver description specifies how to calculate the size of the key context as a function of the key type and size. This is an object with the following properties:

It does not explicitly state or require that, it was just an assumption I had made. If that was an incorrect assumption, I can update again.

Reply:
My understanding is that the "key_context" property is provided as part of the JSON file describing the driver. This information is extracted by the build system from the JSON file and provided somehow to the core to "allow the core to calculate the size of the context from the key parameters in typical use cases". This information is thus not intended to be in a driver key context.

Reply:
Ok, I will update.